### DISCUSSION TITLE
add "System.IO.Directory - CopyDirectory'

### DISCUSSION THREAD

**Discussion by wstaelens** [dotnet/runtime#60960] [Ideas] [answered]
see from 2018: https://github.com/microsoft/dotnet/issues/794

**Comment by gfoidl** [comment]
If you want this API to be added, so please see [API Review Process](https://github.com/dotnet/runtime/blob/main/docs/project/api-review-process.md) and [file an issue](https://github.com/dotnet/runtime/issues/new?assignees=&labels=api-suggestion&template=02_api_proposal.yml&title=%5BAPI+Proposal%5D%3A+) so that it can be handled according that process.

**Comment by Symbai** [comment, marked as answer]
Just one day before you asked I've already filled out such an issue #60903
